watertap/data/techno_economic/zeroth_order_model_data/aeration_basin.yaml

```diff
@@ -0,0 +1,41 @@
+default:
+  default_removal_mass_solute:
+    unit: kg/kg
+    value: 0
+  energy_electric_flow_vol_inlet:
+    unit: kWh/m^3
```

Comment:
It would be better to have powers in Python format (`**`) as it means we can directly parse the units into Pyomo objects. Otherwise, we need to first process the `^` into a `**`.

Reply:
Regarding folders, I think we gonna have a lot of data of different type in that folder, cost data, model performance data (as in this folder) For user navigation, it might make sense to have separate data locations so they are organized at least based on something.  (order of the model, data type?)

Reply:
It might make sense to have things organized based on data type. But I think for now this `techno_economic` folder will be for data for "WaterTAP3-style" models. As we discussed last time, the distinction between "0-order" and "0-dimensional" is not always clear, so I wouldn't pick that as the organization method.

Reply:
I'll also disagree with @andrewlee94 on the power representation. If users are editing this file, `^` is more typical in mathematical type-setting and elsewhere. And its easy enough to replace every instance of `^` with `**`.

Reply:
yeah I agree with @andrewlee94 and @bknueven, lets just drop these into the technoeconomic folder and we can reorganize into subfolders later